gazelle/bzl/gazelle.go

```diff
@@ -324,3 +322,20 @@ func (s srcsList) Contains(m string) bool {
 	}
 	return false
 }
+
+// checkInternalVisibility overrides the given visibility if the package is
+// internal. This is taken from Gazelle's rule package. Unfortunately bzl
+// allows for both the name "internal" and "private" to indicate visibility and
```

Comment:
I don't think `rule.CheckInternalVisibility` necessarily needs to be mentioned here. Arguably that should have been in `language/go`. Each language may have its own definition of internal visibility, so I'd expect each extension might have a function like this.

Reply:
Done